File: meu-teste/Models/Equipment_model_state_hourly_earnings.cs
namespace equipment_model_state_hourly_earnings.Model
{
    public class Equipment_model_state_hourly_earnings
    {
        public int id;
        public Guid Equipment_model_id { get; set; }
        public Guid Equipment_state_id { get; set; }
        public Single Value { get; set; }
    }
}

File: Aiko Teste/teste/src/main/java/TesteEstagioBackendV2/teste/Model/EquipmentModelStateHourlyEarnings.java
package TesteEstagioBackendV2.teste.Model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.UUID;

@Getter
@Setter
@Entity
@AllArgsConstructor
@NoArgsConstructor
public class EquipmentModelStateHourlyEarnings {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private UUID id;

    @ManyToOne(cascade = CascadeType.ALL)
    @JoinColumn(name = "equipment_model_id")
    private EquipmentModel equipmentModel;

    @ManyToOne(cascade = CascadeType.ALL)
    @JoinColumn(name = "equipment_state_id")
    private EquipmentState equipmentState;

    @Column
    private float value;


}
